src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,41 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
         --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
+        --admin-user-enabled -a  : Whether the admin user account is enabled.  Allowed values: false,
```

Comment:
This should be a flag that doesn't require a value.
According to this help, it looks like currently, it would be `--admin-user-enabled false/true` which isn't CLI convention.
/cc @tjprescott